example/layer-examples.js

```diff
@@ -9,140 +9,144 @@ import {
   ArcLayer64,
   ChoroplethLayer64,
   ExtrudedChoroplethLayer64,
-  LineLayer64,
+  LineLayer64
 
-  EnhancedChoroplethLayer
 } from 'deck.gl';
 
+import dataSamples from './data-samples';
+import {flattenCoords} from './utils';
+
 const ArcLayerExample = props =>
   new ArcLayer({
-    ...props,
-    id: props.id || 'arcLayer',
-    data: props.arcs,
-    strokeWidth: props.arcStrokeWidth || 1,
+    id: 'arcLayer',
+    data: props.data,
+    getSourcePosition: f => flattenCoords(f.geometry.coordinates).shift(),
+    getTargetPosition: f => flattenCoords(f.geometry.coordinates).pop(),
+    getSourceColor: f => f.properties.DIRECTION === 'INBOUND' ? [0, 128, 200] : [0, 200, 128],
+    getTargetColor: f => f.properties.DIRECTION === 'INBOUND' ? [0, 200, 128] : [0, 128, 200],
+    strokeWidth: 1,
     pickable: true,
-    onHover: props.onArcHovered,
-    onClick: props.onArcClicked
+    onHover: props.onHovered,
+    onClick: props.onClicked
   });
 
 const ChoroplethLayerContourExample = props =>
   new ChoroplethLayer({
-    ...props,
-    id: props.id || 'choroplethLayerContour',
-    data: props.choropleths,
+    id: 'choroplethLayerContour',
+    data: props.data,
+    getColor: f => [0, 80, 200],
     opacity: 0.8,
     drawContour: true
   });
 
 const ChoroplethLayerExample = props =>
   new ChoroplethLayer({
-    ...props,
-    id: props.id || 'choroplethLayerSolid',
-    data: props.choropleths,
-    opacity: 0.1,
+    id: 'choroplethLayerSolid',
+    data: props.data,
+    getColor: f => [((f.properties.ZIP_CODE * 10) % 127) + 128, 0, 0],
+    opacity: 0.8,
     pickable: true,
-    onHover: props.onChoroplethHovered,
-    onClick: props.onChoroplethClicked
+    onHover: props.onHovered,
+    onClick: props.onClicked
   });
 
 const ScreenGridLayerExample = props =>
   new ScreenGridLayer({
-    ...props,
-    id: props.id || 'screenGridLayer',
-    data: props.points,
-    pickable: false,
-    opacity: 0.06
+    id: 'screenGridLayer',
+    data: props.data,
+    getPosition: d => d.COORDINATES,
+    unitWidth: 40,
+    unitHeight: 40,
+    minColor: [0, 0, 80, 0],
+    maxColor: [100, 255, 0, 128],
+    pickable: false
   });
 
 const LineLayerExample = props =>
```

Comment:
Also sending `modelMatrix`, `onHover` and `onClick`. The extrudedChoroplethLayer uses `mapViewState` for lighting.